js/navigation-search.js

```diff
@@ -54,7 +67,52 @@ bcpl.navigationSearch = (($) => {
 		window.location = `${bcpl.constants.basePageUrl}/search.html?q=${searchTerms}&page=1&resultsPerPage=10`;
 	};
 
-		/**
+	const navBackButtonClicked = (event) => {
```

Comment:
navBackButtonClicked and modalCoverClicked are the same? should this be one method?